pages/admin/players/playerForm/academics.tsx

```diff
@@ -0,0 +1,108 @@
+import { joiResolver } from "@hookform/resolvers/joi";
+import Button from "components/Button";
+import PlayerFormField from "components/PlayerFormField";
+import Joi from "joi";
+import { StateMachineProvider, useStateMachine } from "little-state-machine";
+import { useRouter } from "next/router";
+import React, { useState } from "react";
+import { useForm } from "react-hook-form";
+import updateAction from "utils/updateActionPlayer";
+import DashboardLayout from "components/DashboardLayout";
+import PlayerFormLayout from "components/Player/PlayerFormLayout";
+
+export type AcademicFormValues = {
+  gpa?: string;
+  disciplinaryActions?: string;
+};
+
+const PlayerProfileFormSchema = Joi.object<AcademicFormValues>({
+  gpa: Joi.string().empty("").default(null),
+  disciplinaryActions: Joi.string().empty("").default(null),
+});
+
+const UserSignUpPageOne: React.FC = () => {
+  const router = useRouter();
+
+  // TODO: Add loading state to form submission
+  const [submitting, setSubmitting] = useState(false);
+  const [error, setError] = useState("");
+  const { errors, register, handleSubmit } = useForm<AcademicFormValues>({
+    resolver: joiResolver(PlayerProfileFormSchema),
+  });
+  const { action, state } = useStateMachine(updateAction);
+
+  async function onSubmit(
+    values: AcademicFormValues,
+    event?: React.BaseSyntheticEvent
+  ): Promise<void> {
+    event?.preventDefault();
+    if (submitting) {
+      return;
+    }
+    try {
+      action(values);
+      router.push("/admin/players/playerForm/attendence");
+    } catch (err) {
+      setError(err.message);
+    } finally {
+      setSubmitting(false);
+    }
+  }
+
+  return (
+    <DashboardLayout>
+      <StateMachineProvider>
```

Comment:
I believe there's a StateMachineProvider at the top-level of the app, so you don't have to include another provider here.